src/main/java/com/bzabek/ticketbookingapp/Configuration.java

```diff
@@ -0,0 +1,12 @@
+package com.bzabek.ticketbookingapp;
+
+@org.springframework.context.annotation.Configuration
```

Comment:
dlaczego bez importu?

Reply:
Ponieważ nazwałem klasę tak tak samo jak nazywa się adnotacja, przez to tak to wygląda.